addons/mail/static/src/core/common/navigable_list.js

```diff
@@ -59,6 +59,10 @@ export class NavigableList extends Component {
         return Boolean(this.state.open && (this.props.isLoading || this.props.options.length));
     }
 
+    get sortedOptions() {
+        return this.props.options.sort((o1, o2) => (o1.group ?? -1) - o2.group);
```

Comment:
This is very confusing.

Especially, if o1.group can be undefined, can't o2.group also be?

Using `-` operator when values can be undefined could lead to unexpected results.

Reply:
Agree, the `??` should apply on both groups. Nitpicking but no `group` being `0` is more intuitive. And this LOC is easier to read thank to seeing a single `-` rather than 3